charts/super-agent-deployment/README.md

```diff
@@ -0,0 +1,86 @@
+[![Community Plus header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Community_Plus.png)](https://opensource.newrelic.com/oss-category/#community-plus)
+
+# super-agent-deployment
+
+![Version: 0.0.0-beta](https://img.shields.io/badge/Version-0.0.0--beta-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: TODO-FILL-THIS-WITH-CI](https://img.shields.io/badge/AppVersion-TODO--FILL--THIS--WITH--CI-informational?style=flat-square)
+
+A Helm chart to install New Relic Super agent on Kubernetes
+
+# Helm installation
+
+You can install this chart using directly this Helm repository:
+
+```shell
+helm repo add newrelic https://newrelic.github.io/helm-charts
+helm upgrade --install newrelic/CHART-TEMPLATE -f your-custom-values.yaml
+```
+
+## Values managed globally
+
+This chart implements the [New Relic's common Helm library](https://github.com/newrelic/helm-charts/tree/master/library/common-library) which
+means that it honors a wide range of defaults and globals common to most New Relic Helm charts.
+
+Options that can be defined globally include `affinity`, `nodeSelector`, `tolerations`, `proxy` and others. The full list can be found at
+[user's guide of the common library](https://github.com/newrelic/helm-charts/blob/master/library/common-library/README.md).
+
+## Chart particularities
+
+> **TODO:** Here is where you should add particularities for this chart like what does the chart do with the privileged and
+low data modes or any other quirk that it could have.
+
+### TODOs
+There are values that should be planned at some point of removed from the `values.yaml`. I leave them here documented
```

Comment:
> There are values that should be planned at some point of removed from the `values.yaml`

I would reword this sentence since it is a bit confusing

Reply:
As I documented the TODOs [here](https://github.com/newrelic/helm-charts/pull/1155#discussion_r1365573200) y removed the section from the README on [#0a14139](https://github.com/newrelic/helm-charts/pull/1155/commits/0a141392c9c249476c8ca69630b65908defff74a)